tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/actions/debugger_actions.ts

```diff
@@ -47,3 +50,42 @@ export const debuggerRunsRequestFailed = createAction(
  * Actions for the Alerts Component.
  */
 export const alertsViewLoaded = createAction('[Debugger] Alerts View Loaded');
+
+/**
+ * Actions for the Timeline Component.
+ */
+export const activeRunIdChanged = createAction(
+  '[Debugger] Debugger Active Run Changed',
+  props<{activeRunId: string | null}>()
+);
+
+export const numExecutionsRequested = createAction(
+  '[Debugger] Number of Executions Requested'
+);
+
+export const numExecutionsLoaded = createAction(
+  '[Debugger] Number of Executions Loaded',
+  props<{numExecutions: number}>()
+);
+
+export const requestExecutionDigests = createAction(
```

Comment:
Please fix: this action is fired from the UI and this is not describing user's action onto the view. It is describing how view understood user's interaction and converting it to the system's reaction to it.